how-to/workspace-platform-starter/client/src/framework/bootstrapper.ts

```diff
@@ -65,7 +66,17 @@ export async function init(): Promise<boolean> {
 	let workspaceMetaInfo: RegistrationMetaInfo | undefined;
 	let notificationMetaInfo: RegistrationMetaInfo | undefined;
 
+	await PlatformSplash.updateProgress("Integrations");
+	logger.info("Registering integrations");
+	await registerIntegration(customSettings.integrationProvider, moduleHelpers, async (query) => {
+		if (homeRegistration?.setSearchQuery) {
```

Comment:
Maybe we should log a warning if someone tries to call it before it is ready?

Reply:
I will add that

Reply:
Done